locations/spiders/extended_stay_america.py

```diff
@@ -1,52 +1,16 @@
-# -*- coding: utf-8 -*-
-import json
-import re
+from scrapy.spiders import SitemapSpider
 
-import scrapy
+from locations.structured_data_spider import StructuredDataSpider
+from locations.user_agents import BROSWER_DEFAULT
```

Comment:
> BROSWER

Reply:
Well... fuck

Reply:
I've fixed it here, and I'll put it on my TODO list to fix it in the other spiders